### ISSUE TITLE
IPH Single Owner with 10$/MMBtu strange behavior

### ISSUE THREAD

**Issue by sjanzou** [NREL/SAM#1924]
From @taylorbrown75 

I was running the IPH tower single owner model and noticed strange behavior related to the ppa_price on the Revenues page.

If you set the price to $10/MMBtu, the calculated LCOH is negative, and the IRR is close to 300%.

However, if you set the price lower or higher than $10, the results seem to behave normally.

 

Below is a table of parametric results, with the default IPH tower single owner case where I varied the ppa price from 8-30 $/MMBtu.

![image](https://github.com/user-attachments/assets/98808bb5-b9ea-4ae7-987e-f79c920f2b89)

![image](https://github.com/user-attachments/assets/c2737aa8-ccf5-44d6-b1db-42fdb5c1841e)

**Comment by sjanzou** [comment]
@taylorbrown75, this is actually a DSCR sculpted debt artifact.

If you run the parametrics with 50% debt instead of a DSCR of 1.3
![image](https://github.com/user-attachments/assets/40b87343-d9d0-43a2-b3ca-885392c8f959)

You get the "expected" results:
![image](https://github.com/user-attachments/assets/0d4e0e89-463a-49f2-94ad-39314a1fbe83)

In the default case (DSCR = 1.3) with the parametric runs,
![image](https://github.com/user-attachments/assets/23741d7d-01f4-44aa-8380-6c3e432b14f1)

If you look at the suspect $10/MMBtu case in detail (right click on row header - "3" in the case shown and select "Create new case"
![image](https://github.com/user-attachments/assets/210ae33b-dc1f-438d-9a82-d883adf93933)

You can run and examine the suspect $10/MMBtu case in detail
![image](https://github.com/user-attachments/assets/0efc045f-b205-41cf-9c29-1432311b19c8)

Also, when running the case, you get the simulation messages
![image](https://github.com/user-attachments/assets/4246d3b9-027f-4771-a574-6c5ba11742e2)

And then examination of the Cash flow, shows the suspect "Debt principal payment" line
![image](https://github.com/user-attachments/assets/8d84b890-9ae4-403b-b981-0e3fb4a765bf)

Which leads to the suspect cash flow after tax graph on the "Summary" results page
![image](https://github.com/user-attachments/assets/37b3d552-08af-4aa3-9c0a-f258feba08bc)

In summary, if you run in DSCR more for the debt, always be suspect of 0% "Debt percent" reported in the metrics table
![image](https://github.com/user-attachments/assets/f4067301-ae56-4706-8282-fe9fffc3c2af)

@cpaulgilman - anything to add?

**Comment by sjanzou** [comment]
SAM project file used to create the aforementioned results... 
[SAM_1924.zip](https://github.com/user-attachments/files/17867845/SAM_1924.zip)

**Comment by cpaulgilman** [comment]
@taylorbrown75 I this is an issue with understanding the effect of **Maximum debt fraction** and not a bug. Can you confirm this issue is ok to close?

**Comment by taylorbrown75** [comment]
> [@taylorbrown75](https://github.com/taylorbrown75) I this is an issue with understanding the effect of **Maximum debt fraction** and not a bug. Can you confirm this issue is ok to close?

Yes I agree. This issue is okay to close.
